src/language-markdown/printer-markdown.js

```diff
@@ -790,8 +790,9 @@ function printUrl(url, dangerousCharOrChars = []) {
       ? dangerousCharOrChars
       : [dangerousCharOrChars]),
   ];
+
   return new RegExp(dangerousChars.map((x) => `\\${x}`).join("|")).test(url)
-    ? `<${url}>`
+    ? `<${url.replaceAll("<", "%3C").replaceAll(">", "%3E")}>`
```

Comment:
What do you think use a helper function, 
```js
const encodeUrl = (url, characters) => {
    for (const character of characters) {
        url = url.replaceAll(character, encodeURIComponent(character))
    }
    return url;
}
```

so we don't need hard code `%3C`

Reply:
Sounds great, should I include this function in the util.js file in the same directory or in the file itself?

Reply:
It's fine to put in the file for now.